app/models/trends/query.rb

```diff
@@ -94,6 +94,13 @@ def perform_queries
     to_arel.to_a
   end
 
+  def language_order_for(trend_class)
+    trend_class
```

Comment:
The trends classes all expose a `klass` method which returns the thing that will be  trending (ie, `PreviewCard`). If we wanted to avoid passing in the model class this way, we could have them also expose `trend_klass` or something (which would be `PreviewCardTrend` in that example, and adjusted for others)